tools/flang2/flang2exe/cgmain.cpp

```diff
@@ -1292,6 +1293,38 @@ remove_dead_instrs(void)
   }
 }
 
+/**
+   \brief process debug info of variables with parameter attribute.
```

Comment:
PARAMETERs are constants, not variables.

Reply:
Thanks. Incorporating.